### ISSUE TITLE
NPE in 

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo#12646]
### Describe your problem!

Crash when trying to create a new waypoint, when tapping to open the coordinate input.
This reminds me of #12616 but I am using todays nightly.

### How to reproduce?

- Open cache
- Open waypoint tab
- Create a new waypoint
- Click on coordinate field to start entering coords

### Actual result after these steps?

Crash

```
01-30 21:38:12.499  3023  3023 E AndroidRuntime: FATAL EXCEPTION: main
01-30 21:38:12.499  3023  3023 E AndroidRuntime: Process: cgeo.geocaching, PID: 3023
01-30 21:38:12.499  3023  3023 E AndroidRuntime: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.widget.Button.setVisibility(int)' on a null object reference
01-30 21:38:12.499  3023  3023 E AndroidRuntime: 	at cgeo.geocaching.ui.dialog.CoordinatesInputDialog.onCreateView(CoordinatesInputDialog.java:249)
01-30 21:38:12.499  3023  3023 E AndroidRuntime: 	at androidx.fragment.app.Fragment.performCreateView(Fragment.java:2963)
01-30 21:38:12.499  3023  3023 E AndroidRuntime: 	at androidx.fragment.app.DialogFragment.performCreateView(DialogFragment.java:489)
01-30 21:38:12.499  3023  3023 E AndroidRuntime: 	at androidx.fragment.app.FragmentStateManager.createView(FragmentStateManager.java:518)
01-30 21:38:12.499  3023  3023 E AndroidRuntime: 	at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:282)
01-30 21:38:12.499  3023  3023 E AndroidRuntime: 	at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:2189)
01-30 21:38:12.499  3023  3023 E AndroidRuntime: 	at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:2100)
01-30 21:38:12.499  3023  3023 E AndroidRuntime: 	at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:2002)
01-30 21:38:12.499  3023  3023 E AndroidRuntime: 	at androidx.fragment.app.FragmentManager$5.run(FragmentManager.java:524)
01-30 21:38:12.499  3023  3023 E AndroidRuntime: 	at android.os.Handler.handleCallback(Handler.java:883)
01-30 21:38:12.499  3023  3023 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:100)
01-30 21:38:12.499  3023  3023 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:237)
01-30 21:38:12.499  3023  3023 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8107)
01-30 21:38:12.499  3023  3023 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
01-30 21:38:12.499  3023  3023 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:496)
01-30 21:38:12.499  3023  3023 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1100)
```

### Expected result after these steps?

Normal operation

### Reproducible

Yes

### c:geo Version

2022.01.30-NB

### System information

```text
Samsung Galaxy Tab S4, Android 10
```

### Additional Information

Found on `master`/nightly but should be checked whether also relevant for `release`

**Comment by eddiemuc** [comment]
@Lineflyer I assume this problem only occurs in landscape mode -> correct?

Fixed with PR #12648. Problem also affects release.

**Comment by Lineflyer** [comment]
> @Lineflyer I assume this problem only occurs in landscape mode -> correct?

Correct, therefore I merged your fix.
Only found this testing on my tablet, where landscape is the "default". :)
